homeassistant/components/tts/google.py

```diff
@@ -70,7 +81,7 @@ def async_get_tts_audio(self, message, language=None):
         # If language is not specified or is not supported - use the language
```

Comment:
We should not have these checks.

1. This check should be moved to the service, as every platform will need this check
2. We should not default to another language but instead log an error.